gnomad_qc/v3/subset/subset.py

```diff
@@ -0,0 +1,292 @@
+import argparse
+import logging
+import sys
+
+import hail as hl
+
+from gnomad_qc.v3.resources.meta import meta as metadata
+from gnomad_qc.v3.resources.annotations import get_info
+from gnomad_qc.v3.resources.raw import get_gnomad_v3_mt
+from gnomad.resources.grch38.gnomad import GENOME_POPS
+from gnomad.utils.filtering import subset_samples_and_variants
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s: %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+HEADER_DICT = {
+    "format": {
+        "GQ": {"Description": "Genotype Quality"},
+        "SB": {
+            "Description": "Per-sample component statistics which comprise the Fisher's Exact Test to detect strand bias"
+        },
+        "AD": {
+            "Description": "Allelic depths for the ref and alt alleles in the order listed"
+        },
+        "PID": {
+            "Description": "Physical phasing ID information, where each unique ID within a given sample (but not across samples) connects records within a phasing group"
+        },
+        "GT": {"Description": "Genotype"},
+        "MIN_DP": {"Description": "Minimum DP observed within the GVCF block"},
+        "PGT": {
+            "Description": "Physical phasing haplotype information, describing how the alternate alleles are phased in relation to one another"
+        },
+        "PL": {
+            "Description": "Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification"
+        },
+        "DP": {"Description": "Approximate read depth"},
+        "RGQ": {"Description": ""},
+    },
+    "info": {
+        "AC": {"Description": "Alternate Allele count in gnomAD post-filtering"},
+        "AC_raw": {"Description": "Raw alternate allele count in gnomAD"},
+        "AS_ReadPosRankSum": {
+            "Description": "allele specific Z-score from Wilcoxon rank sum test of each Alt vs. Ref read position bias"
+        },
+        "AS_MQRankSum": {
+            "Description": "Allele-specifc Z-score from Wilcoxon rank sum test of alternate vs. reference read mapping qualities"
+        },
+        "AS_RAW_MQ": {
+            "Description": "Allele-specifc raw root mean square of the mapping quality of reads across all samples"
+        },
+        "AS_QUALapprox": {
+            "Description": "Allele-specifc sum of PL[0] values; used to approximate the QUAL score"
```

Comment:
Most of these definitions were taken from: https://github.com/broadinstitute/gatk/blob/master/src/test/resources/org/broadinstitute/hellbender/tools/walkers/GnarlyGenotyper/twoSampleASDB.vcf. I combine the QUALapprox definition with Allele-specific but I think in this case it might be best to leave it to the default description to avoid confusion?

Reply:
I actually don't see `QUALapprox` or `AS_QUALapprox` in ether of the UKBB or gnomad v3.0 headers. I am fine leaving as is if this is the default then